deploy.sh

```diff
@@ -1,22 +1,31 @@
 #!/bin/bash
 
+#Build linux-version first and determine the build-version
+GOOS=linux GOARCH=amd64 go build -ldflags '-s -w -extldflags "-static"' .
+BUILD_VERSION=$(space-cloud -v | cut -f3 -d ' ')
+
+zip space-cloud_$BUILD_VERSION.zip space-cloud
+mv ./space-cloud_$BUILD_VERSION.zip ./linux/
+rm space-cloud
 mkdir linux && mkdir windows && mkdir darwin
 GOOS=darwin GOARCH=amd64 go build -ldflags '-s -w -extldflags "-static"' .
-zip space-cloud.zip space-cloud
-mv ./space-cloud.zip ./darwin/
+zip space-cloud_$BUILD_VERSION.zip space-cloud
+mv ./space-cloud_$BUILD_VERSION.zip ./darwin/
 rm space-cloud
 GOOS=windows GOARCH=amd64 go build -ldflags '-s -w -extldflags "-static"' .
-zip space-cloud.zip space-cloud.exe
-mv ./space-cloud.zip ./windows/
+zip space-cloud_$BUILD_VERSION.zip space-cloud.exe
+mv ./space-cloud_$BUILD_VERSION.zip ./windows/
 rm space-cloud.exe
-GOOS=linux GOARCH=amd64 go build -ldflags '-s -w -extldflags "-static"' .
-zip space-cloud.zip space-cloud
-mv ./space-cloud.zip ./linux/
-rm space-cloud
 
 # echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
 # docker push spaceuptech/space-cloud:latest
 
+#Upload as versioned build
+curl -H "Authorization: Bearer $JWT_TOKEN" -F 'file=@./darwin/space-cloud_'$BUILD_VERSION'.zip' -F 'fileType=file' -F 'makeAll=false' -F 'path=/darwin' https://spaceuptech.com/v1/api/downloads/files
```

Comment:
The single quotes over here are not required and might give an error:
```
'$BUILD_VERSION'.zip
```

Reply:
If the single quotes are removed like so 
```
curl -H "Authorization: Bearer $JWT_TOKEN" -F 'file=@./darwin/space-cloud_$BUILD_VERSION.zip' -F 'fileType=file' -F 'makeAll=false' -F 'path=/darwin' https://spaceuptech.com/v1/api/downloads/files

```
the variable is not processed and the file `space-cloud_$BUILD_VERSION.zip` is being looked for instead.

Reply:
Okay. changed it to be in double-quotes completely.